paasta_tools/monitoring_tools.py

```diff
@@ -186,36 +203,41 @@ def send_event(service, check_name, overrides, status, output, soa_dir, ttl=None
     if not team:
         return
 
-    runbook = overrides.get('runbook', 'http://y/paasta-troubleshooting')
     system_paasta_config = load_system_paasta_config()
     if cluster is None:
         try:
             cluster = system_paasta_config.get_cluster()
         except PaastaNotConfiguredError:
             cluster = 'localhost'
+
     result_dict = {
+        'name': check_name,
+        'runbook': overrides.get('runbook', 'http://y/paasta-troubleshooting'),
+        'status': status,
+        'output': output,
+        'team': team,
+        'page': get_page(overrides, service, soa_dir),
         'tip': get_tip(overrides, service, soa_dir),
         'notification_email': get_notification_email(overrides, service, soa_dir),
+        'check_every': overrides.get('check_every', '1m'),
+        'realert_every': overrides.get('realert_every', monitoring_defaults('realert_every')),
+        'alert_after': overrides.get('alert_after', '5m'),
         'irc_channels': get_irc_channels(overrides, service, soa_dir),
         'slack_channels': get_slack_channels(overrides, service, soa_dir),
         'ticket': get_ticket(overrides, service, soa_dir),
         'project': get_project(overrides, service, soa_dir),
-        'page': get_page(overrides, service, soa_dir),
-        'alert_after': overrides.get('alert_after', '5m'),
-        'check_every': overrides.get('check_every', '1m'),
-        'realert_every': overrides.get('realert_every', monitoring_defaults('realert_every')),
+        'priority': get_priority(overrides, service, soa_dir),
         'source': 'paasta-%s' % cluster,
+        'tags': get_tags(overrides, service, soa_dir),
         'ttl': ttl,
+        'sensu_host': system_paasta_config.get_sensu_host(),
+        'sensu_port': system_paasta_config.get_sensu_port(),
+        'component': get_component(overrides, service, soa_dir),
+        'description': get_description(overrides, service, soa_dir),
     }
 
-    sensu_host = system_paasta_config.get_sensu_host()
```

Comment:
This is a change in behavior, because we'll could be sending "null" into pysensu-yelp, which means in turn it will try to 'connect' to it, right?

Reply:
I do still check below with `if result_dict.get('sensu_host'):` if this value is set to `None` (or a non-empty value anyway), do you think there should be some more controls or protections on it?